tests/configurable_cluster.py

```diff
@@ -125,21 +128,19 @@ def stop_host(self, host_name):
         # Change the topology to something that doesn't exist
         ips = self.get_ip_address_dict()
         down_hostname = self.get_down_hostname(host_name)
-        self.exec_cmd_on_host(
-            self.master,
-            'sed -i s/%s/%s/g ~/.prestoadmin/config.json' %
-            (host_name, down_hostname)
-        )
-        self.exec_cmd_on_host(
-            self.master,
-            'sed -i s/%s/%s/g ~/.prestoadmin/config.json'
-            % (ips[host_name], down_hostname)
-        )
+        self.exec_cmd_on_host(self.master,
+                              'sed -i s/%s/%s/g %s' %
+                              (host_name, down_hostname, get_config_file_path())
+                              )
```

Comment:
changed it.  (also had to move the first argument down one line to do that, or you get a pep warning about indentation).